### PR TITLE
add switch.mqtt_template documentation

### PR DESCRIPTION
**Pull request by natiz**
**Description:**
See HA PR for details.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13771

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/switch.mqtt_template.markdown [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+This platform is similar to the [switch.mqtt](/components/switch.mqtt) platform, only it allows the usage of a template as the on/off payload.
+
+To enable this switch in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yml entry
+switch:
+  - platform: mqtt_template
+    command_topic: "home/bedroom/switch1/set"
```
Comment (frenck) [inline source/_components/switch.mqtt_template.markdown:26]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thanks, @natiz. I've reviewed the PR and left you a comment. Could you please take a look? 👍

**Comment by natiz** [comment]
Thanks @frenck. Waiting to see if my PR (home-assistant/home-assistant#13771) is gonna be accepted, will then look into that comment.

**Comment by DubhAd** [comment]
I note that the linked PR was closed 10 days ago, closing this too

**Pull request closed without merging**